src/sidebar.ts

```diff
@@ -500,68 +440,93 @@ const identitiesSidebar = [
       "security-model",
       "identities/native-browser",
       "concepts/redirects",
+      {
+        type: "category",
+        label: "User flows",
+        link: {
+          type: "doc",
+          id: "kratos/self-service",
+        },
+        items: [
+          "kratos/concepts/browser-redirect-flow-completion",
+          "kratos/self-service/flows/user-registration",
+          "kratos/self-service/flows/user-login",
+          "kratos/self-service/flows/user-logout",
+          "kratos/self-service/flows/user-settings",
+          "kratos/self-service/flows/verify-email-account-activation",
+          "kratos/self-service/flows/account-recovery-password-reset",
+          "kratos/self-service/flows/user-facing-errors",
+        ],
+      },
     ],
   },
   {
     type: "category",
-    label: "Self-service user flows",
-    collapsed: false,
-    collapsible: false,
-    link: {
-      type: "doc",
-      id: "kratos/self-service",
-    },
-    items: [
-      "kratos/concepts/browser-redirect-flow-completion",
-      "kratos/self-service/flows/user-registration",
-      "kratos/self-service/flows/user-login",
-      "kratos/self-service/flows/user-logout",
-      "kratos/self-service/flows/user-settings",
-      "kratos/self-service/flows/verify-email-account-activation",
-      "kratos/self-service/flows/account-recovery-password-reset",
-      "kratos/self-service/flows/user-facing-errors",
-    ],
-  },
-  {
-    type: "category",
-    label: "Authentication",
+    label: "Guides",
     collapsed: false,
     collapsible: false,
-    link: {
-      type: "doc",
-      id: "kratos/concepts/credentials",
-    },
     items: [
-      "concepts/password-policy",
-      "kratos/passwordless/passwordless",
-      "kratos/mfa/overview",
+      "getting-started/local-development",
+      {
+        type: "category",
+        label: "Authenctication",
+        items: [
+          "getting-started/local-development",
+          "kratos/concepts/credentials/username-email-password",
+          "kratos/passwordless/passkeys",
+          "kratos/passwordless/one-time-code",
+          "kratos/organizations/organizations",
+          "kratos/emails-sms/custom-email-templates",
```

Comment:
the custom email templates doc is different than the others in this category - maybe it makes sense for emails/communication to have their own category? or move it to the "configuration" category?

Reply:
I think the title is what is misleading. The content shares how to setup for account recovery via email, login code, registration code, etc. The title should be something like: "Email notification"